### ISSUE TITLE
Wrong startup project

### ISSUE THREAD

**Issue by sinatrocious** [AvaloniaUI/TemplateStudio#27]
### Describe the bug

After creating a new "Avalonia C# Project` and hitting `F5` there is always an error:

> A project with an Output Type of Class Library cannot be started directly.

because "StartupProject" of solution is set to a `AvaloniaApplicationX` project, which is wrong. It should be set to `AvaloniaApplicationX.Desktop` project.

### To Reproduce

 1. In Visual Studio startup window choose "Create a new project".
 2. In "Create a new project" window select "Avalonia C# Project" (C# - All platforms - Avalonia)
 3. Click next, click create.
 4. In "New Avalonia app (AvaloniaApplicationX)" select only platform "Desktop" and hit create (Design pattern = Reactive UI, Features = none).
 5. Hit `F5`, you should see the message box with error.

### Additional context

It'a very minor issue, but would be nice to have a fix if it's easy to do. I create new projects quite often just to test something isolated from main projects and I need every time to select some file in solution explorer, click `Alt`+`Enter`, select solution, then in Properties change "Startup project", which is quite annoying to do several times a day.

### About your setup

 - Visual Studio Version:  17.7.7
 - Template Studio Wizard Version: 1.3
 - Windows Build: 10.0.22621 Build 22621, Microsoft Windows 11 Pro

**Comment by Wezz19** [comment]
Yes the "*.Desktop" project should be set as "Startup Project" by default, if it's possible. The program should compile and run without any modifications when the user press the "Start Debugging" button or F5. I thought something was wrong with the installation the first few times this happened XD.
